### PR TITLE
footer changed

### PR DESCRIPTION
**Pull request by vosaul**
Changed footer structure end dada file
#5

### CODE REVIEW

**Comment by davidgatti** [comment]
Sitemap is not working + when you make a new branch, make sure to always pull from the Master branch to include the new changes. 

Screen Shot 2020-09-11 at 9 03 04 AM

File: _src/_data/footer.yml [hunk 1]
```diff
@@ -1,75 +1,35 @@
...
-  - title: Changelog
-    link : "#!"
-
-services_title: Services
-services:
-  - title: Documentation
-    link : "#!"
-
-  - title: Changelog
+  - title: OpenVPN Server      <div class="col-12 col-md-4 col-lg-3">
```
Comment (davidgatti) [inline _src/_data/footer.yml:19]: What is this HTML stuff next to the OpenVPN Server?

**Review by davidgatti — COMMENTED** [review]
Check my question.

**Comment by vosaul** [comment]
When I changed the footer many times I checked everything, everything was fine. But when I created the last committee, I was wrong somewhere. I don't know how this could have happened, everything worked locally for me.

I'm sorry, I misunderstood the rules again. I closed this request because I sent a corrected one.

**Comment by davidgatti** [comment]
Screen Shot 2020-09-13 at 4 01 53 PM

Footer still is not working as requested. Please test you code locally!

**Comment by vosaul** [comment]
image

David, sorry I don't know how this could have happened. I checked, everything worked. But after I did the PR, in the branch for some reason the old navbar. 
I once again fixed everything manually, pushed out, and checked each file on GitHub. Now everything should work properly.

**Review by davidgatti — APPROVED** [review]

**Pull request merged**